store/datastore.go

```diff
@@ -4,11 +4,20 @@ import (
 	"context"
 	"time"
 
+	"github.com/ONSdigital/dp-dataset-api/mongo"
+	"github.com/ONSdigital/dp-dataset-api/neo4j"
 	"github.com/gedge/mgo/bson"
 
 	"github.com/ONSdigital/dp-dataset-api/models"
 )
 
+var _ Storer = DatasetStore{}
+
+type DatasetStore struct {
+	*mongo.Mongo
+	*neo4j.Neo4j
+}
+
```

Comment:
The above block of code is in main also? I am guessing we can get rid of the code in main?